src/VarArg.hs

```diff
@@ -23,6 +23,19 @@ class SuperComposition a b c | a b -> c where
     -- True
     --
     -- Inspired by <http://stackoverflow.com/questions/9656797/variadic-compose-function>.
+    --
+    -- ==== Performance
+    -- To check the performance there was done a bunch of benchmarks. Benchmarks were made on
+    -- examples given above and also on a function of many arguments.
+    -- The results are showing that the operator ('...') performs as fast as
+    -- plain applications of the operator ('Prelude..') on all the tests.
```

Comment:
I think this comparison is not good. Looks like `(.)` operator may lead to the same performance draw-down if ghc fails to inline it